ap-roller.py

```diff
@@ -258,7 +259,7 @@ def main(args: "argparse.Namespace"):
     py_args = '-O' if args.optimize else []
     for ap in aps:
         assert os.path.isdir(str(ap)), f"AP directory or venv does not exist: '{ap}'"
-    output_dir = os.path.join("output", str(os.getpid()))
+    output_dir = os.path.join("output", f"{os.getpid()}_{datetime.now().strftime('%m_%d-%H_%M-%S')}")
```

Comment:
I like how you smuggled the dir name change in there :P
Maybe move pid to the back so it's sorted?

Reply:
yeah i realized how annoying it was to check against generations so added this in and just pushed it to the same branch and forgot to change the title lol. will do